### PR TITLE
Join labeler and welcome into single workflow

### PR DESCRIPTION
**Pull request by trsvchn**
Fixes #{issue number}

Description:

This workflow will be triggered only on PR/Issue opened: Greeting step on issue opened + PR opened. Labeling step on PR opened.

The aim is to reduce a number of unnecessary GH actions runs.

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Review by vfdev-5 — APPROVED** [review]
Looks good, thanks

**Review by vfdev-5 — APPROVED** [review]
Thanks @trsvchn

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Request Labeler
+#         if: github.event_name == 'pull_request_target'
+        uses: actions/labeler@v3
+        with:
+          configuration-path: .github/pr-labeler.yml
```
Comment (sdesrozis) [inline .github/workflows/triage.yml:17]: Does `or-labeller.yml` still useful ? Because it has been removed in the PR

Comment (trsvchn) [reply .github/workflows/triage.yml:17]: Yes, workflow file`.github/workflows/pr-labeler.yml` is removed, and action moved to `.github/workflows/triage.yml` but `.github/pr-labeler.yml` is a config for that action, containing info about labels. I think we cannot put invalid workflow yml file inside workflows dir, that's why config file lives there.

File: .github/workflows/triage.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Request Labeler
+#         if: github.event_name == 'pull_request_target'
+        uses: actions/labeler@v3
+        with:
+          configuration-path: .github/pr-labeler-config.yml
```
Comment (vfdev-5) [inline .github/workflows/triage.yml:17]: So, this is our local file right ? But we do not have any file like that ?

Comment (trsvchn) [reply .github/workflows/triage.yml:17]: Now we have
https://github.com/pytorch/ignite/pull/1835/files#diff-c07d414af06346d8cbebb019db1e0079729e57365ed5954a77058bca6c53bb62

Comment (trsvchn) [reply .github/workflows/triage.yml:17]: after discussion https://github.com/pytorch/ignite/pull/1835#discussion_r598448898
we decided to rename it

Comment (vfdev-5) [reply .github/workflows/triage.yml:17]: OK, I see, my bad

**Pull request merged**